### PR TITLE
update faq and opts as talked yesterday?

### PR DESCRIPTION
**Pull request by alexmyczko**

### CODE REVIEW

File: static/faq.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
 Filters file names according to their extension.<br>
 To find source code dealing with XMPP written in Perl, you could search for "<tt>XMPP
 filetype:perl</tt>".<br>
-The currently supported file types are c, c++, perl, python, go, java, ruby, shell, vala, javascript, json.
+The currently supported file types are c, objective-c, objective-c++, c++, perl, python, go, java, ruby, shell, vala, javascript, json.
```
Comment (stapelberg) [inline static/faq.html]: Here you list “objective-c” and “objective-c++”, but in the other file you introduce them as “objc” and “objc++”. I’d prefer the shorter names, so let’s change it to “objc” and “objc++” here.

**Comment by stapelberg** [comment]
Thanks for the updated pull request and sorry for the late reply.

There’s only 2 minor changes to be done before I can merge this.

Let me know if you’d prefer me to do these changes for you.

**Comment by alexmyczko** [comment]
It's certainly faster if you go ahead and just do it(tm). And I'm not sure I can do tabs in the web gui, have to tried some time. Thanks! At least it's now complete with the .mm files for objc++, although that's also not so common as the .m (objc)... Thanks!

**Comment by stapelberg** [comment]
Alright, fixed and pushed the commit for you: https://github.com/Debian/dcs/commit/f33e380eba34e30002ee66c20ba50049f82a4e72 — this change is now live.

Thanks!

**Pull request closed without merging**